sources/packages/web/src/views/aest/student/Applications.vue

```diff
@@ -1,12 +1,10 @@
 <template>
-  <v-card class="mt-4">
-    <div class="mx-5 py-4">
-      <StudentApplications
-        :studentId="studentId"
-        @goToApplication="goToApplication"
-      />
-    </div>
-  </v-card>
+  <tab-container>
+    <StudentApplications
```

Comment:
we can rename it to `student-applications`

Reply:
done